Marlin/src/lcd/menu/menu.cpp

```diff
@@ -90,9 +90,10 @@ void MarlinUI::_goto_previous_screen(TERN_(TURBO_BACK_MENU_ITEM, const bool is_b
       is_back ? 0 : sh.top_line,
       sh.items
     );
-  }
-  else
+  } else {
     return_to_status();
+  }
+  defer_status_screen(false);
```

Comment:
There is no guarantee that the previous screen from any arbitrary screen is not also deferring the status screen. Consider the case where something like a mesh editor has to run a select screen.

So… While this supports the case for screen handlers that want to defer the status screen calling `defer_status_screen` continually, that isn't always the case. So, before we can insert this assumption, we would need to audit all screen handlers that want to retain status-screen deferral to make sure that they are calling `defer_status_screen` themselves on every invocation.